zcash_client_sqlite/CHANGELOG.md

```diff
@@ -25,13 +25,21 @@ and this library adheres to Rust's notion of
 - An `unstable` feature flag; this is added to parts of the API that may change
   in any release. It enables `zcash_client_backend`'s `unstable` feature flag.
 - New summary views that may be directly accessed in the sqlite database.
-  The structure of these views should be considered unstable; they may 
+  The structure of these views should be considered unstable; they may
   be replaced by accessors provided by the data access API at some point
   in the future:
   - `v_transactions`
   - `v_tx_received`
   - `v_tx_sent`
 - `zcash_client_sqlite::wallet::init::WalletMigrationError`
+- A filesystem-backed `BlockSource` implementation
+  `zcash_client_sqlite::FsBlockDb`. This block source expects blocks to be
+  stored on disk in individual files named following the pattern
+  `<blockmeta_root>/blocks/compactblock-<blockheight>-<blockhash>`. A SQLite
```

Comment:
There are some differences here compared to our prior discussion in the [GitHub issue](https://github.com/zcash/librustzcash/issues/591).  Were there additional considerations that came up as you were implementing this?

The implementation should work as-is.  However the primary key is the the blockheight-blockhash, and to me it seems more logical to put those in the front.  The compactblock suffix is more like a filetype.

Thoughts?